src/clj_kondo/impl/analyzer/re_frame.clj

```diff
@@ -9,22 +9,38 @@
 (defn new-id! []
   (str (swap! counter inc)))
 
+(defn- assoc-reg-maybe [ctx name-expr fq-def]
```

Comment:
Perhaps a better name for this function is `with-context` or so, since we replaced "reg" with "context".